docs/events.md

```diff
@@ -0,0 +1,91 @@
+# Caluma Events
+
+Caluma emits events at certain stages of a workflow. These can be used to implement own "side-effects".
+
+In contrast to the `validations` extension, events are emitted AFTER a certain event happened.
+
+## Configuration
+
+To configure a module where event receivers should be imported from, you can set the `EVENT_RECEIVER_MODULES` environment variable.
+
+When using the Caluma service, you can place your receivers in `caluma.extensions.events` and set `EVENT_RECEIVER_MODULES` to
+`["caluma.extensions.events"]`.
+
+## Example event receiver
+
+```python
+from caluma.caluma_core.events import on
+from caluma.caluma_workflow.events import complete_work_item
+from caluma.caluma_workflow.serializers import CompleteWorkItemSerializer
+
+
+@on(complete_work_item)
+def send_mail_on_complete_work_item(sender, work_item, **kwargs):
+    if work_item.task.slug == "slug-we-are-interested-in":
+        # send notification
+        pass
+
+
+# It's also possible to specify a sender:
+@on(complete_work_item, sender=CompleteWorkItemSerializer)
+def send_mail_on_complete_work_item_2(sender, work_item, **kwargs):
+    if work_item.task.slug == "slug-we-are-interested-in":
+        # send notification
+        pass
+```
+
+## @on
+
+The `@on` decorator takes two arguments:
```

Comment:
Please add a note on why we're doing the `@on` decorator instead of using Django's `@receiver`

Reply:
Done.